src/lib.rs

```diff
@@ -30,7 +30,7 @@ mod upconversion;
 
 pub use error::Error;
 pub use fontinfo::FontInfo;
-pub use glyph::{Glyph, GlyphName};
+pub use glyph::{pen::Pen, Glyph, GlyphName};
```

Comment:
Is this something that consumers of norad need to access directly?

Reply:
My idea is that Pen is the official thing you build Glyphs with programmatically (and have some correctness checks), instead of constructing things manually.